<launch>
	<group ns="flea">	
		<include file = "$(find pointgrey_camera_driver)/launch/camera.launch">
			<arg name="calibrated" value ="1"/>
			<arg name="camera_serial" value= "flea"/>
		</include>
	</group>
	<node pkg = "confidence_models" type ="manual_pattern_frame.sh" name ="world_frame"/>
	<node name ="overlay_2D_manual" pkg="confidence_models" type="identification_manual" output ="screen">
		<param name = "roi_l" value = "1"/>
		<param name = "roi_r" value = "1"/>
		<param name = "roi_u" value = "1"/>
		<param name = "roi_b" value = "1"/>
		<param name = "show_markers" value = "true"/>
	</node>
	<node name = "gui" pkg ="rqt_gui" type ="rqt_gui"/>
</launch>